writings-volume-1/updated/02-chapter-1.adoc

```diff
@@ -359,6 +358,7 @@ Isaac Penington
 
 [.signed-section-context-close]
 Aylesbury Jail, 15th of 3rd month, 1667
+//Does this above need to be embedded within this testimony?
```

Comment:
Yes, In this chapter there are two long testimonies of Penington about his own journey. But since there is also a narrator (Thomas Ellwood in this case) popping in and out, I think it would be easiest for the reader to distinguish the difference between who was talking if we embedded these two testimonies (using `[.embedded-content-document.testimony]`. This one starts up several pages with the phrase: “A true and faithful relation, in brief, concerning myself,..."